### PR TITLE
Fixed TypeError when no proxies are enabled

### PR DESCRIPTION
**Pull request by nestorsokil**

### CODE REVIEW

File: demo/payment.py [hunk 1]
```diff
@@ -122,11 +122,14 @@ def _charge(payload, url=None):
         root_url = current_app.config['VGS_PROCESSOR_ROOT_URL']
         url = urljoin(root_url, '/charge')
 
-    proxy_setting = ProxySetting.query.filter(ProxySetting.active == True).first()
+    proxy_setting = ProxySetting.query.filter(ProxySetting.active is True).first()
```
Comment (mahmoudimus) [inline demo/payment.py]: no, this is wrong.

File: demo/payment.py [hunk 2]
```diff
@@ -122,11 +122,14 @@ def _charge(payload, url=None):
...
     if not proxy_setting and ProxySetting.proxy_env_variables_present(current_app.config):
         proxy_setting = ProxySetting.from_config(current_app.config)
 
-    proxies = proxy_setting.as_dict()
+    if proxy_setting is not None:
```
Comment (mahmoudimus) [inline demo/payment.py]: ah this is a valid bug. good catch, make it more pythonic plz :)

```python
proxies = proxy_setting.as_dict() if proxy_setting else None
```

File: demo/payment.py (hunk 1 above)
Comment (nestorsokil) [reply demo/payment.py]: Yeah, you're right, sqlalchemy magic doesn't work this way, sorry.

File: demo/payment.py (hunk 2 above)
Comment (nestorsokil) [reply demo/payment.py]: sure, thanks

**Comment by mahmoudimus** [comment]
:shipit: 🌟 great job @nestorsokil !

**Pull request merged**